src/SourceBuild/content/eng/pipelines/templates/jobs/sdk-diff-tests.yml

```diff
@@ -165,3 +169,12 @@ jobs:
       mergeTestResults: true
       publishRunAttachments: true
       testRunTitle: $(Agent.JobName)
+
+  - ${{ if and(eq(parameters.publishTestResultsPr, 'true'), or(eq(variables['Build.SourceBranch'], 'refs/heads/main'), startsWith(variables['Build.SourceBranch'], 'refs/heads/release'))) }}:
+      - template: ../steps/create-baseline-update-pr.yml
+        parameters:
+          pipeline: sdk
+          repo: dotnet/sdk
+          originalTestResultsPath: src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets
+          updatedTestResultsPath: $(Build.StagingDirectory)/BuildLogs
+          pullRequestTitle: Update SDK Diff Tests Baselines and Exclusions
```

Comment:
I think source-build should be included in the title.  There are UB SDK diff tests as well.